.github/workflows/pytest-workflow.yml

```diff
@@ -142,8 +144,17 @@ jobs:
           conda install -n base conda-libmamba-solver
           conda config --set solver libmamba
 
+      # Set up secrets
+      - name: Use secrets
+        env:
+          SENTIEON_LICENSE_BASE64: ${{ secrets.SENTIEON_LICENSE_BASE64 }}
```

Comment:
Is `${{ secrets.SENTIEON_LICENSE_BASE64 }}` referring to a GitHub-secret?

Is it strictly necessary to set the env variable `SENTIEON_LICENSE_BASE64`?  Couldn't one just do

`run: nextflow secrets set SENTIEON_LICENSE_BASE64 ${{ secrets.SENTIEON_LICENSE_BASE64 }}`

Reply:
probably, let me try that.
I think I even had that in a previous version, but it was before I figured I needed to make a PR not from a fork...

Reply:
Done